images/linux/scripts/installers/aws-sam-cli.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+################################################################################
+##  File:  aws-sam-cli.sh
+##  Desc:  Installs AWS SAM CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install aws sam cli
+brew tap aws/tap
+brew install aws-sam-cli
```

Comment:
Since installation of AWS-Sam-CLI is pretty small, I would suggest to add this logic to `aws.sh` file instead of creating the new one. So all tools related to AWS will be in the single place:
https://github.com/actions/virtual-environments/blob/master/images/linux/scripts/installers/aws.sh